stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -799,7 +808,7 @@ extension _TensorComputation {
 // user code, so they are generally just wrappers around the implementation
 // above.
 
-/// Loads the TF computation from a binary TF FunctionDef proto given by 'bytes'
+/// Load the TF computation from a binary TF FunctionDef proto given by 'bytes'
```

Comment:
Plural verbs in comments were intentional - Swift code usually uses plural verbs. If you see other parts of the `TensorFlow` module and the standard library, they are plurals too.